README.md

```diff
@@ -501,6 +501,54 @@ colord("#008080").mix("#808000", 0.35).toHex(); // "#50805d"
 
 </details>
 
+<details>
+  <summary><b><code>.tints(colors = 5)</code></b> (<b>mix</b> plugin)</summary>
```

Comment:
Let's maybe call the param count to make it clear that the user should pass a number here (not a color)?